components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa/src/Publisher/Models/OpcNodeOnEndpointModel.cs

```diff
@@ -0,0 +1,46 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.IIoT.OpcUa.Publisher.Models {
+    using System;
+    using Newtonsoft.Json;
+    using System.Text;
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// Class describing a list of nodes
+    /// </summary>
+    [DataContract]
+    public class OpcNodeOnEndpointModel {
```

Comment:
`DataSetFieldId` and `SkipFirst` properties are missing when compared to `OpcNodeModel` class.

Reply:
added